packages/babel-plugin-emotion/src/index.js

```diff
@@ -228,13 +228,50 @@ export function replaceCssObjectCallExpression(path, identifier, t) {
 
 const visited = Symbol('visited')
 
+const parseImports = (node, currentNames) => {
+  const names = Object.assign({}, currentNames)
+  const imports = node.specifiers.filter(
+    specifier =>
+      specifier.type === 'ImportDefaultSpecifier' ||
+      specifier.type === 'ImportSpecifier'
+  )
+
+  imports.forEach(singleImport => {
+    if (singleImport.imported) {
+      // Is helper method
+      names[singleImport.imported.name] = singleImport.local.name
+    } else {
+      // Is default import
+      names.default = singleImport.local.name
+    }
+  })
+
+  return names
+}
 export default function(babel) {
   const { types: t } = babel
 
+  let importedNames = {
```

Comment:
The `importedNames` arg values come from `state.opts`. We might also want to set the imported names to `state.importedNames`